pkg/chain/ethereum/bitcoin_difficulty.go

```diff
@@ -187,7 +188,12 @@ func (bdc *BitcoinDifficultyChain) RetargetWithRefund(headers []*bitcoin.BlockHe
 	}
 
 	// Update Bitcoin difficulty via LightRelayMaintainerProxy.
-	_, err := bdc.lightRelayMaintainerProxy.Retarget(serializedHeaders)
+	_, err := bdc.lightRelayMaintainerProxy.Retarget(
```

Comment:
I only increased the limit for LightRelayMaintainerProxy's retarget function.
I left the LightRelay's retarget function unchanged as it's possible the function could succeed with the default gas limit.